extra-libs/libc+wasi/autobuild/defines

```diff
@@ -5,4 +5,5 @@ BUILDDEP="llvm"
 
 USECLANG=1
 NOSTATIC=0
+ABSTRIP=0
```

Comment:
Did you explain this in commit msg? Resolve if yes.

Reply:
> Did you explain this in commit msg? Resolve if yes.

No, but this will bug out the current version of `binutils` (because the contents are not ELF files).